code/win32/win_syscon.c

```diff
@@ -237,14 +234,12 @@ static LRESULT WINAPI ConWndProc( HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lP
 		{
 			if ( s_timePolarity & 1 )
 			{
-				SetBkColor( ( HDC ) wParam, ERROR_BG_COLOR );
-				SetTextColor( ( HDC ) wParam, ERROR_COLOR_1 );
-			}
-			else
-			{
-				SetBkColor( ( HDC ) wParam, ERROR_BG_COLOR );
-				SetTextColor( ( HDC ) wParam, ERROR_COLOR_2 );
+				SetBkColor( ( HDC ) wParam, EDIT_COLOR );
+				SetTextColor( ( HDC ) wParam, TEXT_COLOR );
+				return ( LRESULT ) s_wcd.hbrEditBackground;
```

Comment:
I'm a bit unsure about returning a same brush handle as the edit box (to change the background color), but it seems to work properly.